app/src/controllers/pages/selectors.js

```diff
@@ -54,6 +54,8 @@ export const pluginsTabSelector = (state) => payloadSelector(state).pluginsTab;
 export const pluginPageSelector = (state) => payloadSelector(state).pluginPage;
 export const pluginRouteSelector = (state) => payloadSelector(state).pluginRoute;
 
+export const adminProfileRouteSelector = (state) => payloadSelector(state).profileRoute;
```

Comment:
As it is not related only for admin, I suggest to rename it to `userProfileRouteSelector`.